package.json

```diff
@@ -26,10 +26,12 @@
   },
   "homepage": "https://github.com/dwyl/elm-datepicker#readme",
   "devDependencies": {
-    "cypress": "^2.1.0"
+    "cypress": "^2.1.0",
+    "elm-format": "0.8.2",
+    "elm-test": "0.19.1-revision2"
   },
   "dependencies": {
-    "elm": "^0.18.0",
-    "http-server": "^0.11.1"
+    "elm": "0.19.1-3",
```

Comment:
These are technically `devDependencies` too given that they are not required by the Date Picker when used in an `Elm` project. 💭